log-sink/src/test/java/org/springframework/cloud/stream/module/log/LogSinkApplicationTests.java

```diff
@@ -39,11 +52,39 @@
 	@Bindings(LogSink.class)
 	private Sink sink;
 
+	@SuppressWarnings("unused")
 	@Autowired
 	private Sink same;
 
+	@Autowired
+	private LoggingHandler logSinkHandler;
+
+	@BeforeClass
+	public static void setUp() {
```

Comment:
Sorry, just looking at this now.
This `PropertiesInitializer` stuff should be replaced by `@[Web]IntegrationText({"name=foo", etc...})` IMHO